consensus/state.go

```diff
@@ -683,11 +686,13 @@ func (cs *ConsensusState) handleTimeout(ti timeoutInfo, rs cstypes.RoundState) {
 
 }
 
-func (cs *ConsensusState) handleTxsAvailable(height int64) {
+func (cs *ConsensusState) handleTxsAvailable() {
 	cs.mtx.Lock()
 	defer cs.mtx.Unlock()
 	// we only need to do this for round 0
-	cs.enterPropose(height, 0)
+	heightToPropose := cs.Height
```

Comment:
no need to create a separate variable

Reply:
Idea here was being paranoid, as there's still time passing between the call to cs.Height in the logging statement and the actual passing. (and I'm coming from Java where we usually don't access fields but methods, so multiple calls should be consolidated into one variable.
Will remove this.